bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
+    locale = l10n_utils.get_locale(request)
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
+    if lang_file_is_active(new_home, locale):
```

Comment:
At first we were using the same code we're using for the [home page](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L426), but it doesn't work because of `firefox/os/devices`, which is enabled for most locales at this point.

I double checked to be sure with `DEV = False`and xh. The page loads the new template, but the main file is not activated, while /devices is.

The last snippet would work.